proposed/2023/PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,47 @@
+# Markdown Readme Rendering
+* Start Date: 6/27/23
+* Authors: Allie Barry ([albarry4](https://github.com/albarry4))
+* Issue: https://github.com/NuGet/Home/issues/12583
+
+## Summary
+
+Support rendering markdown readmes in the package details page for NuGet in Visual Studio to make package affordances more accesible for customers browsing packages within the PMUI, and to increase motivation for high quality Readme inclusion for package authors. This will be part of a larger effort to enhance the experience around Readmes and increase their inclusion rate across packages in the NuGet ecosystem. 
+
+## Motivation
+
+In the latest NuGet in Visual Studio 17.5 customer survey, our team found that one of the largest problems that our customers face while using NuGet in the Visual Studio ecosystem is around understanding package affordances – specifically calling out the usage of package metadata such as READMEs. Only 35.35% of survey respondents feel that they can adequately understand the affordances of a package (does it do what I need to to do when I need it?) while browsing and installing packages. Including a detailed README file and implementing features to facilitate README viewing and inclusion would positively impact this number.   
+
+## Explanation
+
+### Functional Explanation
```

Comment:
This section should outline details about how the experience will work. Currently, the document is missing details that are relevant to the UI in Visual Studio, such as consideration for installed packages, multiple package sources, and packages with README.txt files.

A few questions that come to mind that this section should answer as part of the functional spec (probably not an exhaustive list, happy to collaborate on ideas, but these are the ones that come to mind immediately):

1. Will README files only be rendered for packages in the Browse tab?
2. Will we show a README for installed packages in the Installed tab? If so, it would be worth calling out where we will be getting this README from. I would imagine we'd be pulling it in from disk rather than from the remote package source.
3. Will we show README for packages in the Updates tab?
5. Will we show README markdown for the Solution-level package manager or just Project-level package manager?
6. What will we do in cases where there's a README.txt file for an installed package? Will we display that as well in the details pane or is this experience only for README if it's in markdown format?
7. What will we show in the details pane if there is no README for the selected package?